Added New "Lights" Category + Organized More Items

PepperTheVixen opened the pull request:
updates khanshoaib3/stardew-access#439

## Changelog

### Tile Tracker Changes

- Added new "Lights" category for lighting items such as braziers, fires, and fireplaces.
- Moved beds into "Interactables".
- Organized a bunch more items into the various categories.
- Removed custom names of geodes and resource nodes (coal node, iron node, etc.) as SV v1.6 added names for these in the game itself.

PepperTheVixen committed:
Added new "Lights" category for lighting items and organized more items.

part of khanshoaib3/stardew-access#439

khanshoaib3 reviewed (commented):
So moving barrels and item boxes into the empty dictionary makes their names internationalised right? I wonder if the geodes have gotten a name as well, can you check that? Then we can also move them into the same empty dictionary, removing the need i18n them.

PepperTheVixen committed:
added the anvil and mini forge to the machines category

khanshoaib3 commented on stardew-access/assets/TileData/QualifiedItemIds.json:
```diff
@@ -148,34 +226,94 @@
+      "(O)82",
+      "(O)86"
     ],
     "Frozen geode": [
```
Can we also move this to the empty dictionary? That'll remove the i18n overhead for these.

ParadoxiKat commented:
Needs fluent token to translate the category name added in i18n/en.ftl.
Can search for other category names in that file and see how they're done. Just make a new entry that says Lighting.
Make sure to add the name of the new key under the translation section in the change log so translators can take note. Might have to edit your original comment to slip that in.

PepperTheVixen committed:
added new categories to i18n

PepperTheVixen commented:
Beep.

I added translation keys for the new categories.
Also, this should fix #455

khanshoaib3 committed:
feat(item): Removed custom names of some items (nodes and geodes)

SV v1.6 added names for many more items, these items are one of them, so I'm just removing the custom names for less translation overhead.

khanshoaib3 committed:
docs: Add changelogs from #446 to latest.md
